.github/workflows/ci.yml

```diff
@@ -588,3 +588,24 @@ jobs:
         if: steps.web-change.outputs.run == 'true'
         run: npm run web:release
         working-directory: oxidation/client
+
+  ##############################################################################
+  #                            🏁 Windows icon overlays                       #
+  ##############################################################################
+
+  windows-icon-overlay:
```

Comment:
Do we really want to include the build in the CI :thinking:

Reply:
That's what @Max-7 asked.

Reply:
Just for the packaging, not for every PR.